experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -437,22 +479,30 @@ function proj_of_image_ideal(G::LinearlyReductiveGroup, rep_mat::MatElem)
     W = image_ideal(G, rep_mat)
     mixed_ring_xy = base_ring(W[2])
     n = ncols(rep_mat)
-    m = G.group[2]
+    if isdefined(G, :group)
+        m = (group_dim(G))^2
+    else
+        m = ngens(base_ring(group_ideal(G)))
+    end
     #use parallelised groebner bases here. This is the bottleneck!
-    return eliminate(W[1], gens(mixed_ring_xy)[(2*n)+1:(2*n)+(m^2)]), W[2]
+    return eliminate(W[1], gens(mixed_ring_xy)[(2*n)+1:(2*n)+(m)]), W[2]
```

Comment:
Does this also work? In that case you can get rid of the computation of `m` above. Also the same then could be done in `generators` a few lines below.
```suggestion
    return eliminate(W[1], gens(mixed_ring_xy)[(2*n)+1:end]), W[2]
```